app/controllers/api/v1/organizations_controller.rb

```diff
@@ -4,8 +4,8 @@ class OrganizationsController < ApiController
       include Api::OrganizationsController
       before_action :find_organization, only: %i[users listings articles]
       before_action :authenticate!, only: %i[create update destroy]
-      before_action :authorize_admin, only: %i[create update]
-      before_action :authorize_super_admin, only: %i[destroy]
+      before_action :authorize_admin, only: %i[update]
+      before_action :authorize_super_admin, only: %i[create destroy]
```

Comment:
I think we'll ultimately want this to indeed be any site admin (not org admin, but site admin) to be consistent with reactions — _but_ I'm cool with us rolling with this exactly as you have it here and circling back for a convo after we've shipped this to scrutinize whether we have everything right where we want it.

Given that we made an adjustment to handle the delete action in a certain way with the other PR, let's touch base to make some final judgments before moving on.